src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Header {
-
+  logo = 'Conduit logo';
 };
 
 class PageObject {
+  constructor(url, header) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  commentButton = '[Publish comment] button';
```

Comment:
To create a getter property for `commentButton`, use the `get` keyword before the method name. For example: `get commentButton() { return '[Publish comment] button'; }`. 🌟